tests/unit/mock/config/parser/nokia_sros/sros_full_sent.txt

```diff
@@ -0,0 +1,5562 @@
+exit all
```

Comment:
Does it make sense to include this in the example? I am a bit torn (if I understand this correctly), what is the expected result, but we should be purposeful in our decision.

Reply:
I was simply copying this example from @h4ndzdatm0ld TTP repo with the assumption this is a standard format for configurations.

Reply:
@itdependsnetworks @jdrew82 I am happy to share a ContainerLab setup with a Nokia SROS device for the team to use for current and future testing or integrations, etc. If interested, please let me know. Also, the basic 'show config' (admin display-config) equivalent of SROS will always include `exit all` at the end.

Reply:
Also, on the needing more configs to test, I could dig up some old configs but they will need some sanitizing.

Reply:
that would be great.